tests/src/test/scala/system/packages/MessageHubFeedTests.scala

```diff
@@ -274,6 +274,67 @@ class MessageHubFeedTests
       assert(matchingActivations.length == 0)
   }
 
+  it should "return correct status and configuration" in withAssetCleaner(wskprops) {
+    val currentTime = s"${System.currentTimeMillis}"
+
+    (wp, assetHelper) =>
+      val triggerName = s"/_/dummyMessageHubTrigger-$currentTime"
+      println(s"Creating trigger ${triggerName}")
+
+      val username = kafkaUtils.getAsJson("user")
+      val password = kafkaUtils.getAsJson("password")
+      val admin_url = kafkaUtils.getAsJson("kafka_admin_url")
+      val brokers = kafkaUtils.getAsJson("brokers")
+
+      createTrigger(assetHelper, triggerName, parameters = Map(
+        "user" -> username,
+        "password" -> password,
+        "api_key" -> kafkaUtils.getAsJson("api_key"),
+        "kafka_admin_url" -> admin_url,
+        "kafka_brokers_sasl" -> brokers,
+        "topic" -> topic.toJson,
+        "isBinaryKey" -> false.toJson,
+        "isBinaryValue" -> false.toJson
+      ))
+
+      val run = wsk.action.invoke(actionName, parameters = Map(
```

Comment:
Should have a test to ensure this fails with an bad auth key.

Reply:
shouldn't that be a test for testing the action itself? i.e. failing with a bad auth key is functionality that is inherent to invoking the action itself, not invoking the action with the parameter `lifecycleEvent` set to `READ`. i'm all for having this test if it doesn't exist for the provider already. but a separate issue should be opened as it doesn't specifically relate to this functionality. i.e. an action should never be able to be invoked without proper authKey

Reply:
I mean the authKey you are passing as a parameter to the action.

Reply:
right. that is a required parameter for the action itself regardless of the value of `lifecycleEvent`

Reply:
I understand that the trigger will not be fetched on the read based on how the OpenWhisk infrastructure is setup. However, is there a test to make sure this provider errors as expected?

Reply:
https://github.com/apache/incubator-openwhisk-package-kafka/issues/219